style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:wght@300;400;600;700;900&family=Roboto:wght@500;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* the header section */
+header {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  padding: 25px;
+  width: 376px;
+  margin: auto;
+}
+
+#logo {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+}
+
+/* headline section */
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 25px 145px;
+  background-image: url(./Images/Group.png);
+  background-position: 294px 380px;
+  background-repeat: no-repeat;
+  width: 376px;
+  margin: auto;
+}
```

Comment:
- [ ] As requested by the previous reviewer, the hamburger icon and background image must be visible on the required minimum mobile width of `375px` but I realize in order to see everything on your page we need to scroll horizontally. Kindly make sure to remove this error which is causing the overflow horizontally. so your website contents must all be correctly visible on a minimum width of `375`px without scrolling.

  - ### YOUR APP BEHAVIOUR ON WIDTH OF `375PX`
  
  ![chrome-capture-2023-3-27 (4)](https://user-images.githubusercontent.com/87197412/234847247-ef87bfae-a5fa-4eeb-9d6c-8eab8388084f.gif)

  - ### FIGMA DESIGN ON WIDTH OF `375PX`
  ![image](https://user-images.githubusercontent.com/87197412/234847224-f236af12-c591-4909-9de8-6bb98e5d18dd.png)